internal/blockchain/ethereum/besu/besu_provider_test.go

```diff
@@ -0,0 +1,48 @@
+package besu
+
+import (
+	"context"
+	"testing"
+
+	"github.com/hyperledger/firefly-cli/pkg/types"
+	"github.com/hyperledger/firefly-common/pkg/fftypes"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewBesuProvider(t *testing.T) {
+	var ctx context.Context
+	testCases := []struct {
+		Name  string
+		Ctx   context.Context
+		Stack *types.Stack
+	}{
+		{
+			Name: "testcase1",
+			Ctx:  ctx,
+			Stack: &types.Stack{
+				Name:                   "TestBesuProviderWithEthconnect",
+				Members:                []*types.Organization{{OrgName: "Org1"}},
+				BlockchainProvider:     fftypes.FFEnumValue("BlockchainProvider", "EthConnect"),
+				BlockchainConnector:    fftypes.FFEnumValue("BlockchainConnector", "EthConnect"),
+				BlockchainNodeProvider: fftypes.FFEnumValue("BlockchainNodeProvider", "EthConnect"),
```

Comment:
Should this be `EthConnect` here? `EthConnect` is a `BlockchainConnector` not a `BlockchainNodeProvider`